app/src/main/res/values/strings.xml

```diff
@@ -181,4 +183,42 @@
     <string name="do_not_sent_button">Do Not Send</string>
     <!-- This string labels a button that when clicked allows the crash data to be sent to Mozilla. -->
     <string name="send_data_button">Send Data</string>
+    <!-- This string labels a button to open the Brightness mode selector-->
+    <string name="brightness_mode_tooltip">Brightness</string>
+    <!-- This string is displayed when selecting a background brightness mode -->
+    <string name="brightness_mode_normal">Normal</string>
+    <!-- This string is displayed when selecting a background brightness mode -->
+    <string name="brightness_mode_dark">Dark</string>
+    <!-- This string is displayed when selecting a background brightness mode -->
+    <string name="brightness_mode_void">Void</string>
+    <!-- This string labels a button to open the Video projection mode selector-->
+    <string name="video_mode_tooltip">Video Projection</string>
+    <!-- This string is displayed when selecting a video playback mode -->
+    <string name="video_mode_3d_side">3D Side by Side</string>
+    <!-- This string is displayed when selecting a video playback mode -->
+    <string name="video_mode_360">360</string>
```

Comment:
Shouldn't numbers be in the non_I10n?

Reply:
I wasn't sure all the languages use `180` or `360` to refer to that kind of videos.

Reply:
numerals can be localised, so yeah we should keep. a better L10n note might say `360` refers to `360°` (or `360-degree` videos)